packages/app/src/app/overmind/namespaces/dashboard/utils.ts

```diff
@@ -1,6 +1,10 @@
 import { SidebarCollectionDashboardFragment as Collection } from 'app/graphql/types';
 import { DELETE_ME_COLLECTION } from './state';
 
+export const isRepoPage = location.pathname.includes('/repositories');
```

Comment:
This doesn't work when you switch pages, and I think it's in general not a good practice to rely on URL string operations to determine what should be done. For example, if there's a directory in the dashboard called "repositories" this could break, and there are probably more examples where it doesn't work.

It would be better if we'd pass in the actions in overmind what the page is that we're dealing with, so we can base the logic on that instead.

Reply:
All done :)